src/redux/books/booksSlice.js

```diff
@@ -11,11 +12,12 @@ const booksSlice = createSlice({
   reducers: {
     addBook: (state, action) => {
       const newBook = action.payload;
-      state.books = state.books.append(newBook);
+      console.log(newBook);
```

Comment:
- [ ] For security purpose, let's kindly always get rid of this link after we are done testing :+1:

Reply:
Thank You @FrederickMih